### PR TITLE
{tools}[intel/2019b] worker v1.6.11

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/1d54634e615c8dacfc6baab3d3945cc8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2536.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1d974361aa980b444de7fb8f37d32172 for a full test report.

File: easybuild/easyconfigs/w/worker/worker-1.6.11-intel-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        'checksums': ['0a9bf977b8aabe76343e88095d2296c8a422410fd2a05a1901f2b20e2e1f6fad'],
+    }),
+    ('IO::Stringy', '2.111', {
+        'source_tmpl': 'IO-stringy-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DS/DSKOLL'],
+        'checksums': ['8c67fd6608c3c4e74f7324f1404a856c331dbf48d9deda6aaa8296ea41bf199d'],
+    }),
+    ('Text::CSV', '2.00', {
+        'source_tmpl': 'Text-CSV-%(version)s.tar.gz',
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/I/IS/ISHIGAKI'],
```
Comment (verdurin) [inline easybuild/easyconfigs/w/worker/worker-1.6.11-intel-2019b.eb]: @boegel Please fix this URL

Comment (boegel) [reply easybuild/easyconfigs/w/worker/worker-1.6.11-intel-2019b.eb]: fixed in 89bb79b

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node361.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/f4aa22544ea727f147ced62402060358 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**